cluster/sync.sh

```diff
@@ -41,13 +41,13 @@ function consistently {
     done
 }
 
-function isOk {
-    desiredNumberScheduled=$(getDesiredNumberScheduled $1)
-    numberAvailable=$(getNumberAvailable $1)
+function isHandlerOk {
```

Comment:
Just a note related to next PR after this, checking handler daemonsets has to be done at operator itself after apply and notify it using the `Status` CRD part with some `Conditions` following the k8s standards, so we will just monitor NMState CRD status.